### ISSUE TITLE
Query timeout limit reached

### ISSUE THREAD

**Issue by tabbassidaloii** [sec2pri/mapping_preprocessing#9]
@DeniseSl22 
I got a timeout error running [geneproteinHumanRedirects query](https://github.com/sec2pri/mapping_preprocessing/blob/main/datasources/Wikidata/queries/geneproteinHumanRedirects.rq).
Shall we try to improve it or divide it into a series of queries?

**Comment by DeniseSl22** [comment]
Yes, it's a known issue for Wikidata, we could execute the GitHub Actions early in the morning in the EU timezone.

**Comment by DeniseSl22** [comment]
Since we're using the Github Action, I haven't received any timeouts. I will add a statement in there that the job should continue, even if there is a timeout (using [always](https://docs.github.com/en/actions/learn-github-actions/expressions#always) statement) to make the rest of the queries more robust.

**Comment by jmillanacosta** [comment]
Will this be solved by #37 ? @DeniseSl22

**Comment by tabbassidaloii** [comment]
@DeniseSl22 do we have the new queries to avoid time out?

**Comment by DeniseSl22** [comment]
Queries are now running on Qlever.
